packages/block-editor/src/components/block-tools/index.js

```diff
@@ -134,3 +122,14 @@ export default function BlockTools( {
 		</div>
 	);
 }
+
+/**
+ * Renders block tools (the block toolbar, select/navigation mode toolbar, the
+ * insertion point and a slot for the inline rich text toolbar). Must be wrapped
+ * around the block content and editor styles wrapper or iframe.
+ *
+ * @param {Object} $0                      Props.
+ * @param {Object} $0.children             The block content and style container.
+ * @param {Object} $0.__unstableContentRef Ref holding the content scroll container.
+ */
```

Comment:
Because it otherwise won't auto generate the md doc files. This seems like a bug in the way we build docs. Cc @nosolosw.

Reply:
The docs are only generated for the entities that are exported. Wrappers such as `forwardRef` are at odds with this approach: what the wrapper is doing with its parameters is something that escapes the auto-documentation code.

It's been a couple of years since I was involved in generating docs, so not sure how common a pain point is this. If it's common, one path we could explore is the use of `@augment` / `@extends` [tags](https://jsdoc.app/tags-augments.html) for these cases. The outer function, `forwardRef`, would use it to point to the docs it wants to expose, the `BlockTools` in this case.

Reply:
A quick search shows 48 occurrences of `export default forwardRef` across the codebase, most of them in block-editor and components packages. The treatment is super inconsistent: a very few have a doc comment just above the forwardRef, some have it above the main function, others have no comment at all. I personally prefer having it above the function as that's what it refers to, and it's most helpful for quick reference when looking at the parameters. But ultimately anything works, as long as we stick to the same rule everywhere 😅

Reply:
I would prefer having it there as well, but it would remove the entry from the docs. Either we have to move it or duplicate it.

Reply:
I think it's best to leave it for now; there are other instances of the same so it can be fixed as a separate issue.